iterator/combined_iterator.go

```diff
@@ -79,7 +79,7 @@ func NewCombinedIterator(
 			IteratorType:      position.TypeCDC,
 			Time:              now,
 			AfterSnapshot:     true,
-			SequenceNumberMap: map[string]string{},
+			SequenceNumberMap: make(map[string]string),
```

Comment:
this change is essentially equivalent, both initialize the map?

Reply:
lol you're right